src/Cloud/weglide/WeGlideSettings.hpp

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#ifndef XCSOAR_CLOUD_WEGLIDE_WEGLIDESETTINGS_HPP
+#define XCSOAR_CLOUD_WEGLIDE_WEGLIDESETTINGS_HPP
+
+#include "util/StaticString.hxx"
+#include "util/TriState.hpp"
+#include "time/BrokenDate.hpp"
+
+#include <cstdint>
+#include <tchar.h>
+
+#define WEGLIDE_PILOT_ID_AS_TEXT  0
+
+/**
+ * WeGlide settings
+ */
+struct WeGlideSettings {
+  /**WeGlide communication
+   */
+  TriState enabled;
```

Comment:
Why TriState?

Reply:
To distinguish whether it is set by user or in a predefined state

Reply:
And why that? If the user didn't enable it, why does it matter whether this was an explicit decision or just the default?

Reply:
Ok, I change this to bool, but for me the TriState make less sense in every case?